docs/sources/guides/intro-to-grafana.md

```diff
@@ -0,0 +1,97 @@
+
+## Grafana process
+
+This is the general process of running Grafana.
+
+1. [Install Grafana on your system]({{< relref "../installation/_index.md" >}}) or [configure]({{< relref "../installation/configure-docker.md" >}}) and [run a Docker image]({{< relref "../installation/docker.md" >}}).
+1. [Add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). (Repeat for each [data source]({{< relref "../features/datasources/data-sources.md" >}}).)
+1. Create a [dashboard]({{< relref "../features/dashboard/dashboards.md" >}}).
+1. Add [panels]({{< relref "../features/panels/panels.md" >}}) to the dashboard.
+
+After creating a dashboard, there are many possible things you might do next. It all depends on your needs and your use case. 
+
+For example, if you want to view weather data and statistics about your smart home, then you might create a playlist. If you are the administrator for a corporation and are managing Grafana for multiple teams, then you might need to set up provisioning and authentication.
+
+The following sections provide an overview of things you might want to do with your Grafana database and links so you can learn more. For more guidance and ideas, check out the [Grafana Community forums](https://community.grafana.com/).
+
+## Explore metrics and logs
+
+Explore your data through ad-hoc queries and dynamic drilldown. Split view and compare different time ranges, queries and data sources side by side. Refer to [Explore]({{< relref "../features/explore/index.md" >}}) for more information.
+
+## Alerts
+
+If you're using Grafana alerting, then you can have alerts sent through a number of different [alert notifiers]({{< relref "../alerting/notifications.md" >}}), including PagerDuty, SMS, email, VictorOps, OpsGenie, or Slack. 
+
+Alert hooks allow you to create different notifiers with a bit of code if you prefer some other channels of communication. Visually define [alert rules]({{< relref "../alerting/rules.md" >}}) for your most important metrics.
+
+## Annotations
+
+Annotate graphs with rich events from different data sources. Hover over events to see the full event metadata and tags.
+
+This feature, which shows up as a graph marker in Grafana, is useful for correlating data in case something goes wrong. You can create the annotations manually—just control-click on a graph and input some text—or you can fetch data from any data source. Refer to [Annotations]({{< relref "../reference/annotations.md" >}}) for more information.
+
+## Dashboard variables
+
+[Variables]({{< relref "../reference/templating.md" >}}), also called templates, allow you to create dashboards that can be reused for lots of different use cases. Values aren't hard-coded with these templates, so for instance, if you have a production server and a test server, you can use the same dashboard for both. 
```

Comment:
I used "variables" because while we have a topic about variables (templating.md), we don't talk much about templating in the docs. It is only mentioned in some of the data source topics.

Reply:
This is a good point. I would say that everything around template variables is confusing - both our UI and our docs. What's most logical for new users?

A template is a known concept but a dashboard doesn't really look like something like a Word template so maybe we should stay away from that wording. On the other hand, looking at Datadog and Chronograf they seem to have followed our lead and call them `template variables` so maybe it is too established to change now?

So in summary, to be on the safe side, I would have `[Template variables]({{< relref "../reference/templating.md" >}}) allow you...`